docs/readmes/development/minikube_deployment.md

```diff
@@ -0,0 +1,356 @@
+      dashboards:
+        emptyDir: {}
+      grafanaData:
+        emptyDir: {}
+
+
+logging:
+  enabled: false
+nginx:
+  secret:
+    certs: orc8r-secrets-secrets-certs
+  create: true
+  replicas: 1
+  podDisruptionBudget:
+    enabled: true
+
+  service:
+    name: bootstrapper-orc8r-nginx-proxy
+    type: LoadBalancer
+
+  image:
+    repository: <DOCKER_REGISTRY>/nginx
+    tag: <TAG>
+    pullPolicy: IfNotPresent
+
+  spec:
+    hostname: orc8r-nginx-proxy.svc.cluster.local
+```
+
+Save this file wherever you want.
+
+### Install with Helm
+
+```
+$ helm dep update
+$ cd MAGMA_ROOT/orc8r/cloud/helm/orc8r
+$ helm install orc8r --namespace magma . --values=<path-to-values-file>
+```
+
+If everything worked you should get something like this:
+
+```
+$ kubectl -n magma get pods
+
+NAME                                             READY   STATUS    RESTARTS   AGE
+mysql-57955549d5-n69pd                           1/1     Running   0          6m14s
+nms-magmalte-7c84667c4c-pvtlj                    1/1     Running   0          2m58s
+nms-nginx-proxy-5b86f479f7-lvjpn                 1/1     Running   0          2m58s
+orc8r-alertmanager-57d5d6ccc4-ht4n4              1/1     Running   0          2m58s
+orc8r-alertmanager-configurer-76cf8f8f57-rmwjf   1/1     Running   0          2m58s
+orc8r-controller-fdf59f456-vvqr2                 1/1     Running   0          2m58s
+orc8r-nginx-7d6c78647-n6d8z                      1/1     Running   0          2m58s
+orc8r-prometheus-77dccb799b-w9z6z                1/1     Running   0          2m58s
+orc8r-prometheus-cache-6d647df4d9-2wqkc          1/1     Running   0          2m58s
+orc8r-prometheus-configurer-6d6d987c88-pgm87     1/1     Running   0          2m58s
+orc8r-user-grafana-6498bb6959-rchx5              1/1     Running   0          2m58s
+postgresql-0                                     1/1     Running   4          6d23h
+```
+
+### Access API
+
+Create an admin user:
+
+```
+kubectl exec -it -n magma \
+    $(kubectl get pod -n magma -l app.kubernetes.io/component=controller -o jsonpath="{.items[0].metadata.name}") -- \
+    /var/opt/magma/bin/accessc add-existing -admin -cert /var/opt/magma/certs/admin_operator.pem admin_operator
+```
+
+Now make sure that the API and your certs are working:
+
+```
+# Tab 1
+kubectl -n magma port-forward $(kubectl -n magma get pods -l app.kubernetes.io/component=nginx-proxy -o jsonpath="{.items[0].metadata.name}") 9443:9443
+```
+
+```
+# Tab 2
+$ curl -k \
+  --cert MAGMA_ROOT/orc8r/cloud/helm/orc8r/charts/secrets/.secrets/certs/admin_operator.pem \
+  --key MAGMA_ROOT/orc8r/cloud/helm/orc8r/charts/secrets/.secrets/certs/admin_operator.key.pem \
+https://localhost:9443
+"hello"
+```
+
+### Access NMS
+
+Create org and user in NMS:
+
+```
+kubectl exec -n magma \
+  $(kubectl get pod -n magma -l app.kubernetes.io/component=magmalte -o jsonpath="{.items[0].metadata.name}") -- \
+  yarn setAdminPassword magma-test admin@magma.test password1234
+```
```

Comment:
x-ref the corresponding portion of the deployment docs for these sections as well, and just keep the nginx port-forwarding part in this document